src/Controls/src/Core/IndicatorView.cs

```diff
@@ -107,25 +108,51 @@ public IEnumerable ItemsSource
 			set => SetValue(ItemsSourceProperty, value);
 		}
 
-
 		protected override SizeRequest OnMeasure(double widthConstraint, double heightConstraint)
 		{
 			var baseRequest = base.OnMeasure(widthConstraint, heightConstraint);
 
 			if (IndicatorTemplate != null)
 				return baseRequest;
 
-			var defaultSize = IndicatorSize + DefaultPadding + DefaultPadding + 1;
-			var items = Count;
+			var padding = DefaultPadding;
+#if __IOS__
+			padding += 7;
```

Comment:
Maybe it makes more sense to apply this in the iOS handler?

Reply:
Yeah. Is it possible to put this into the native code? So the xplat OnMeasure call into that rather? What is the result of the `7`? Seems magical.